AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/bookings-list/bookings-list.component.spec.ts

```diff
@@ -1055,38 +921,6 @@ describe('BookingsListComponent', () => {
         expect(component.showSearch).toBe(false);
     });
 
-    it('should not load venues when search feature is disabled', fakeAsync(() => {
-        referenceDataServiceSpy.getCourts.calls.reset();
-        launchDarklyServiceSpy.getFlag.withArgs(FeatureFlags.adminSearch).and.returnValue(of(false));
-
-        component.subscribeToFeatureFlags();
-        tick();
-
-        expect(component.enableSearchFeature).toBeFalsy();
-    }));
-
-    it('should load venues when search feature is enabled', () => {
```

Comment:
similar to the previous comment, we should ideally keep the tests for the feature being enabled, but remove the code that toggles the feature flag

Reply:
done